src/Textarea.js

```diff
@@ -27,7 +35,7 @@ class Textarea extends Component {
           }}
           onChange={onChange}
           id={computedId}
-          className="materialize-textarea"
+          className={cx('materialize-textarea', className)}
           {...other}
```

Comment:
one thing here @onluiz, could we first add `{...other}` and then overwrite with other props?
```jsx
{...other}
onChange={onChange}
id={computedId}
className={cx('materialize-textarea', className)}
```

Reply:
Done!